postgresql.go

```diff
@@ -62,13 +62,11 @@ func (h *PostgreSQL) tableNames(q queryable) ([]string, error) {
 	var tables []string
 
 	sql := `
-	         SELECT pg_namespace.nspname || '.' || pg_class.relname
-		 FROM pg_class
-		 INNER JOIN pg_namespace ON pg_namespace.oid = pg_class.relnamespace
+	   SELECT DISTINCT g.table_schema || '.' || pg_class.relname
```

Comment:
Changed indentation. Also, is `DISTINCT` necessary here?

Reply:
Sorry for the wrong indentation, there's something wrong with my new code editor...

About the `DISTINCT`, the sql query statement can be replaced as 
```sql
SELECT g.table_schema || '.' || pg_class.relname FROM pg_class JOIN information_schema.role_table_grants g ON pg_class.relname = g.table_name WHERE pg_class.relkind = 'r' AND g.table_schema NOT IN ('pg_catalog', 'information_schema') AND g.privilege_type = 'SELECT';
```

But my concern is that postgresql's fine-grained grants(SELECT/INSERT/UPDATE/DELETE/TRUNCATE etc.). If we just check the `SELECT` permission when `Load()`, things may go wrong later. Do you have some ideas how to trade off this thing?